### DISCUSSION TITLE
convert markdown text into triple quoted string instead of a lot of "\n"?

### DISCUSSION THREAD

**Discussion by retsyo** [marimo-team/marimo#1679] [Q&A]
currently, `marimo convert my.ipynb > my.py` transalte markdown paragraph into a string with a lot of `\n`, for example
```python
mo.md("\n    hello\n    world")
```
as a result, it is hard for user to edit the raw markdown text

So, my question is, anyway to get 
```python
mo.md("'
    hello
    world
"')
```
 
thanks

**Comment by mscolnick** [comment]
hey @retsyo - this seems like it would improve the readability. i like the idea. would you be open to making the contribution? we can walk you through where that code is

**Reply by Abhinavcode13** [reply]
Hi there, @retsyo Maybe I should give this a try.
